docs/source/developer_documentation/rocky.md

```diff
@@ -1,78 +1,127 @@
 
-### Running Rocky
+## Installation
 
-#### Containerized
+### Containerized
 
-To run rocky from the docker container, from the parent directory `nl-kat-coordination`, just run:
+To run rocky from using Docker, run this from the parent directory `nl-kat-coordination`:
 
 ```bash
 $ make kat
 ```
 
 and continue reading this document at "First run".
 
-#### Locally
+### Local
 
-To run rocky locally, follow these steps.
+For a local set up, you need to start the Django app and compile the frontend.
 
-### Installation
+#### Django App
 
-Yarn is used to bundle CSS and Javascript.
-You can build Rocky locally using:
+This requires a working python (>3.10) environment.
+One example of how to create, activate and initialize a development environment is:
+```bash
+$ python3 -m venv $PWD/.venv
+$ source .venv/bin/activate
+$ python3 -m pip install -r requirements-dev.txt
+```
 
+Copy the `.env-dist` to a `.env` and configure the hosts and credentials to PostgreSQL, RabbitMQ and the other services.
 ```bash
-$ make build
+$ cp .env-dist .env
+```
+
+For instance, to configure the PostgreSQL database set the following variables:
+```
+ROCKY_DB_HOST=
+ROCKY_DB_PORT=
+ROCKY_DB=
+ROCKY_DB_USER=
+ROCKY_DB_PASSWORD=
+ROCKY_DB_DSN=
 ```
 
-This will set up Django and compile the frontend.
+Here, `ROCKY_DB_DSN` is optional (e.g. `postgresql://username:password@hostname:port/database_name`)
+and if not set, the other DB variables will be used.
+
 
-#### Running
+Once your environment variables are set up (see `.env-dist`, you can initialize Rocky using:
 
-You can run Rocky using:
+```bash
+$ make build-rocky-native
+```
+
+To start the Django server, run:
 
 ```bash
 $ make run
 ```
 
-#### First run
 
-After running the first time, visit [localhost:8000](http://localhost:8000) in your browser.
-Log in with credentials: admin / admin
+#### Frontend
+
+Yarn is used to bundle CSS and Javascript.
+
+To compile the frontend using yarn locally, run:
+```bash
+$ yarn --ignore-engine
+$ yarn build
+```
 
-You will be prompted to create secure your account with a One Time Password, so get your authenticator ready.
+To compile the frontend using Docker, run:
+```bash
+$ make build-rocky-frontend
+```
 
-#### Testing
+The app should be running at [localhost:8000](http://localhost:8000).
 
-To run all tests, run:
+#### TL;DR
+Given a proper `.env` file, run:
 
 ```bash
-$ make test
+$ python3 -m venv $PWD/.venv
+$ source .venv/bin/activate
+$ python3 -m pip install -r requirements-dev.txt
+$ make build-rocky-native
+$ & make run
+$ make build-rocky-frontend
 ```
 
-#### Database
+## Development
 
-To connect to the PostgreSQL database, set the following environment variables (e.g. "localhost", "5432" etc.):
 
+### Testing
+
+To run all unit tests, run:
+
+```bash
+$ make utest
 ```
-ROCKY_DB_HOST=
-ROCKY_DB_PORT=
-ROCKY_DB=
-ROCKY_DB_USER=
-ROCKY_DB_PASSWORD=
-ROCKY_DB_DSN=
+
+#### Tip
+A local python environment is useful for unit testing even when using Docker.
```

Comment:
```suggestion
A local Python environment is useful for unit testing even when using Docker.
```

Reply:
I solved this in several other places as well :+1: